Makefile

```diff
@@ -131,19 +131,21 @@ init-package: ## Create the zarf init package (must `brew install coreutils` on
 release-init-package:
 	$(ZARF_BIN) package create -o build -a $(ARCH) --set AGENT_IMAGE_TAG=$(AGENT_IMAGE_TAG) --confirm .
 
-# INTERNAL: used to build an iron bank version of the init package with an ib version of the registry image
-ib-init-package:
-	@test -s $(ZARF_BIN) || $(MAKE) build-cli
-	$(ZARF_BIN) package create -o build -a $(ARCH) --confirm . \
-		--set REGISTRY_IMAGE_DOMAIN="registry1.dso.mil/" \
-		--set REGISTRY_IMAGE="ironbank/opensource/docker/registry-v2" \
-		--set REGISTRY_IMAGE_TAG="2.8.3"
-
 # INTERNAL: used to publish the init package
 publish-init-package:
 	$(ZARF_BIN) package publish build/zarf-init-$(ARCH)-$(CLI_VERSION).tar.zst oci://$(REPOSITORY_URL)
 	$(ZARF_BIN) package publish . oci://$(REPOSITORY_URL)
 
+# INTERNAL: used to build an iron bank version of the init package and output to a local directory for testing
+build-registry1-init-package:
+	@test -s $(ZARF_BIN) || $(MAKE) build-cli
+	ZARF_CONFIG=$(ZARF_CONFIG) $(ZARF_BIN) package create --output build --confirm .
+
+# INTERNAL: used to build a release version of the ib init package with a specific agent image
```

Comment:
The changes to the registry chart and image in this PR are valuable as it relates to #375.

On the other hand, I do not think the Zarf team should be publishing and maintaining a registry1 flavored init package because that is a specific use case that can be solved with users building custom init packages:

https://docs.zarf.dev/tutorials/8-custom-init-packages/

We currently only publish and maintain the core Zarf init package, and I do not see a good reason for the Zarf team to maintain any custom init packages for registry1. I would like to see any registry1 specific additions in this PR removed.

Reply:
Will go ahead and remove the registry1 stuff and move to our own repo for publishing custom init package